account_invoice_margin_sale/models/account_invoice.py

```diff
@@ -15,6 +15,7 @@ def _get_margin_applicable_lines(self):
 class AccountMoveLine(models.Model):
     _inherit = "account.move.line"
 
+    # pylint: disable=W8110
```

Comment:
This shouldn't be needed, as you are calling super.

Reply:
This is necessary because currently the super is only called on some lines.

Another possibility is to call the super at the beginning and then update the margin of the lines as needed.

What do you think about this?

Reply:
Yeah, this is on purpose, but the linter fails with this as is?

Reply:
This comment is now necessary to ensure that the linter does not fail.